orchagent/aclorch.cpp

```diff
@@ -1409,10 +1409,6 @@ sai_status_t AclOrch::createBindAclTable(AclTable &aclTable, sai_object_id_t &ta
     attr.value.s32 = SAI_ACL_STAGE_INGRESS;
     table_attrs.push_back(attr);
 
-    attr.id = SAI_ACL_TABLE_GROUP_MEMBER_ATTR_PRIORITY;
```

Comment:
This is an attribute for table group member, not for table. It will act as table size if we use this enum.